tests/by-util/test_pgrep.rs

```diff
@@ -347,3 +347,23 @@ fn test_parent_non_matching_parent() {
         .code_is(1)
         .no_output();
 }
+
+#[test]
+#[cfg(target_os = "linux")]
+fn test_require_handler() {
+    new_ucmd!()
+        .arg("--require-handler")
+        .arg("--signal=INT")
+        .arg("NONEXISTENT")
+        .fails();
```

Comment:
A detail: I would also ensure that there is no output.
```suggestion
        .fails()
        .no_output();
```

Reply:
Good point, added.